services/tests/test_activation.py

```diff
@@ -89,11 +88,57 @@ def test_activate_user_success_for_enterprise_pr_billing(
         dbsession.commit()
         assert user.ownerid in org.plan_activated_users
 
+    def test_activate_user_success_user_org_overlap(
```

Comment:
I'm not sure I understand this test case.

I looks like you are creating 2 orgs with 5 seats taken each. Idk how many total seats this installation (self-hosted) can have. But it seems that all owners of the 2 orgs are distinct (e.g. with different IDs)

So the total count of seats used is 10, and you create a new owner to prove that we can extend that.

Is that the idea?


------
(assuming I'm correct - which I might very well not be)
What I'd like to see:
1. Call `get_installation_plan_activated_users` explicitly to see what users are activated (before and after creating the new user)
2. Have some overlap on the plan_activated_users across the orgs (cause that was the original problem, right?)